docs/client/jdbc/kyuubi_jdbc.rst

```diff
@@ -17,14 +17,14 @@ Kyuubi Hive JDBC Driver
 =======================
 
 .. versionadded:: 1.4.0
-   Since 1.4.0, kyuubi community maintains a forked hive jdbc driver module and provides both shaded and non-shaded packages.
+   Kyuubi community maintains a forked Hive JDBC driver module and provides both shaded and non-shaded packages.
 
-This packages aims to support some missing functionalities of the original hive jdbc.
-For kyuubi engines that support multiple catalogs, it provides meta APIs for better support.
-The behaviors of the original hive jdbc have remained.
+This packages aims to support some missing functionalities of the original Hive JDBC driver.
+For Kyuubi engines that support multiple catalogs, it provides meta APIs for better support.
```

Comment:
Do we have to force them to use capitalized forms?

Reply:
I think it's a common practice, for instance
<img width="938" alt="image" src="https://user-images.githubusercontent.com/26535726/229395814-59a81aee-e6c7-4a56-b348-c6b67f87cbfb.png">

Reply:
OK make sense